package.json

```diff
@@ -171,6 +171,7 @@
     "react-map-gl": "5.2.7",
     "react-medium-editor": "1.8.1",
     "react-responsive": "8.1.0",
+    "react-use-websocket": "2.0.3-b.4",
```

Comment:
Did a cursory look at different options and nothing seemed to be at very established state. Not too tied to this specific one either.